src/components/AssetManagementPanel/AssetManagementForm/FormVariants/ActionSelectionForm/ActionSelectionForm.tsx

```diff
@@ -28,6 +35,39 @@ export const ActionSelectionForm = ({
 }: ActionSelectionFormProps) => {
   const canManage = canSurrender;
 
+  const { setOverlayContent, setOverlayVisible } = useContext(OverlayContext);
+  const onNoAccessHandle = () => {
```

Comment:
The naming a bit weird.... either 

onNoAccess or handleNoAccess

in this case, the later seemed more correct.